src/routes/resources/parts-of-a-shinai-and-how-to-hold-it/+page.svelte

```diff
@@ -0,0 +1,256 @@
+<svelte:head>
+	<title>Parts of a Shinai and How to Hold It | Kendo Club at the University of Michigan</title>
+	<meta
+		name="description"
+		content="Includes labelled diagrams of the bamboo swords used in Kendo and demonstrates how to hold one properly with pictures from multiple angles."
+	/>
+</svelte:head>
+
+<h1>Parts of a Shinai and How to Hold It</h1>
+
+<h2>Anatomy of a Shinai</h2>
+
+<div class="grid-container">
+	<figure>
+		<enhanced:img
+			src="./deconstructed-shinai.jpg?w=400;350;200"
+			sizes="(min-width:1280px) 400px,(min-width:450px) 350px, (min-width:0px) 200px"
+			alt="In black and white, parts of a shinai laid out on the floor."
+		/>
+		<figcaption>A deconstructed shinai</figcaption>
+	</figure>
+
+	<p>
+		A shinai is generally comprised of four slats of bamboo, a leather cap, a plastic plug, a thin
+		leather strap, a handle, a very small square of very thin steel, twine, a cross guard, and the
+		cross guard support. As you can tell from the age of the picture used, the overall design and
+		components of the shinai have not changed dramatically over the years. Although familiarity with
+		the shinai is not required for promotion testing, it is a very important safety factor in your
+		Kendo practice. We request that every member take the time to become familiar with how to take
+		apart, adjust, and reassemble the shinai. This enables you to repair or replace faulty or
+		damaged pieces. A damaged shinai is not only a bad reflection of you as a kendo practitioner,
+		but also a potentially serious threat to your fellow senpai and kouhai. Ask one of your senpai
+		how to take it apart and inspect the pieces. You will eventually want to take the shinai apart
+		so that you can adjust the cord as the leather handle begins to stretch. This is also an
+		excellent opportunity to adjust the handle length if needed.
+	</p>
+</div>
+
+<p>
+	Here are some terms that refer to parts of a shinai. If you become familiar with them, you'll have
+	a common reference for discussion which can greatly reduce confusion in conversation.
+</p>
+
+<div class="grid-container">
+	<figure>
+		<enhanced:img
+			src="./shinai-diagram-japanese-labels.jpg?w=400;350;200"
+			sizes="(min-width:1280px) 400px,(min-width:450px) 350px, (min-width:0px) 200px"
+			alt="Diagram of two shinai labelled in japanese."
+		/>
+		<figcaption>
+			First shinai has its "striking side" facing down. Second shinai has it facing left.
+		</figcaption>
+	</figure>
+
+	<div>
+		<p>Kissaki — Thrusting surface of the shinai. Tip of the sword.</p>
```

Comment:
maybe it would be good to include the alternate spelling to match the picture to allow people who can't read japanese to understand the diagram

Reply:
Yes good point. Added in [264060f](https://github.com/kendoclub-at-umich/website/pull/199/commits/264060fbf1bd5a9a8ceee190a7070964cb7f8643)